src/app/account/PageResetPasswordRequest.tsx

```diff
@@ -84,7 +84,7 @@ export const PageResetPasswordRequest = () => {
           <Center mt="8">
             <Button
               as={RouterLink}
-              to="/login"
+              to="login"
```

Comment:
This is not working ;)
The slash is needed

Reply:
Ah yes, fixed some but forgot this one, thanks 👍🏻